core-bundle/assets/styles/backend-search/container.pcss

```diff
@@ -53,7 +53,10 @@
         left: 0;
         height: 0;
         width: 0;
-        transition: width 0ms .25s, height 0ms .25s, opacity .25s;
+        transition:
+            width 0ms 0.25s,
+            height 0ms 0.25s,
+            opacity 0.25s;
```

Comment:
I'm not sure about that. What is the advantage of spreading this over three lines instead of one? And why is `0.25s` better than `.25s`?

Reply:
> I'm not sure about that. What is the advantage of spreading this over three lines instead of one?

I think better readability as you can see at one glance for which properties a transition is set.

Reply:
Okay, I understand that. But not the `0.25s` change.

Reply:
That rule is opinionated (like most of the things), but I don't think it can be configured. Consistency is more important than what it actually is IMHO.